src/vue/Components/Options/Accounts.vue

```diff
@@ -29,7 +30,14 @@
 
         computed: {
             serverOptions() {
-                let options = [];
+                let options = [
+                    {
+                        id: undefined,
+                        disabled: true,
+                        label: "Please select"
+                    }
+                ];
```

Comment:
What does this option do? You should never be able to select no default server as this is not supported.
Also please use the localisation service to translate the label.